eslint-plugin-console/.eslintrc

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
I'm curious about the reasoning behind removing this package? Just because you're transitioning it to the PF based eslint config?